pkg/scheduler/framework/statement.go

```diff
@@ -254,8 +271,9 @@ func (s *Statement) Allocate(task *api.TaskInfo, hostname string) error {
 	// Update status in session
 	klog.V(3).Info("Allocating operations ...")
 	s.operations = append(s.operations, operation{
-		name: "allocate",
-		args: []interface{}{task, hostname},
+		name:   Allocate,
+		task:   task,
+		reason: hostname,
```

Comment:
allcate reason is hostname?

Reply:
Removed. Actually only for evict it is used, for other ops useless.